pallets/dmp-queue/src/lib.rs

```diff
@@ -250,30 +256,41 @@ pub mod pallet {
 			limit: Weight,
 			_sent_at: RelayBlockNumber,
 			mut data: &[u8],
-		) -> Result<Weight, (MessageId, Weight)> {
-			let message_id = sp_io::hashing::blake2_256(data);
+		) -> Result<Weight, (XcmHash, XcmHash, Weight)> {
```

Comment:
At least a comment on what these return values are. While it would be actually more better to have some `struct` with named parameters.

Reply:
Done.